azure-pipelines.yml

```diff
@@ -352,25 +352,34 @@ stages:
             LTA-89:
               PRODUCT: "SonarQube"
               SQ_VERSION: "LATEST_RELEASE[8.9]"
-              DOTNET_VERSION: "8.22.0.31243"  # Bundled version with SQ 8.9
-              CFAMILY_VERSION: "6.20.5.49286" # Bundled version with SQ 8.9
-              XML_VERSION: "2.2.0.2973"       # Bundled version with SQ 8.9
+              DOTNET_VERSION: "8.22.0.31243"    # Bundled version with SQ 8.9
+              CFAMILY_VERSION: "6.20.5.49286"   # Bundled version with SQ 8.9
+              XML_VERSION: "2.2.0.2973"         # Bundled version with SQ 8.9
+              CSS_VERSION: "1.4.2.2002"         # Bundled version with SQ 8.9
+              JAVASCRIPT_VERSION: "7.4.4.15624" # Bundled version with SQ 8.9
+              PLSQL_VERSION: "3.6.1.3873"       # Bundled version with SQ 8.9
               MSBUILD_PATH: $(MSBUILD_17_PATH)
               TEST_INCLUDE: "**/sonarqube/*"
             LTA-99:
               PRODUCT: "SonarQube"
               SQ_VERSION: "LATEST_RELEASE[9.9]"
-              DOTNET_VERSION: "8.51.0.59060"  # Bundled version with SQ 9.9
-              CFAMILY_VERSION: "6.41.0.60884" # Bundled version with SQ 9.9
-              XML_VERSION: "2.7.0.3820"       # Bundled version with SQ 9.9
+              DOTNET_VERSION: "8.51.0.59060"     # Bundled version with SQ 9.9
+              CFAMILY_VERSION: "6.41.0.60884"    # Bundled version with SQ 9.9
+              XML_VERSION: "2.7.0.3820"          # Bundled version with SQ 9.9
+              CSS_VERSION: "LATEST_RELEASE"      # No official release in SQ 9.9
+              # JAVASCRIPT_VERSION: "9.13.0.20537" # Version 9.13.0.20537 bundled with SQ 9.9, but incompatible with Orchestrator. Use default instead.
+              PLSQL_VERSION: "3.8.0.4948"        # Bundled version with SQ 9.9
               MSBUILD_PATH: $(MSBUILD_17_PATH)
               TEST_INCLUDE: "**/sonarqube/*"
             LTA-2025:
               PRODUCT: "SonarQube"
               SQ_VERSION: "LATEST_RELEASE[2025.1]"
-              DOTNET_VERSION: "10.4.0.108396" # Bundled version with SQ 2025.1
-              CFAMILY_VERSION: "6.62.0.78645" # Bundled version with SQ 2025.1
-              XML_VERSION: "2.12.0.5749"      # Bundled version with SQ 2025.1
+              DOTNET_VERSION: "10.4.0.108396"     # Bundled version with SQ 2025.1
+              CFAMILY_VERSION: "6.62.0.78645"     # Bundled version with SQ 2025.1
+              XML_VERSION: "2.12.0.5749"          # Bundled version with SQ 2025.1
+              CSS_VERSION: "LATEST_RELEASE"       # No official release in SQ 2025.1
```

Comment:
CSS is bundled inside javascript-plugin now, so this adds the plugin twice, no? Once inside javascript, and once LATEST_RELEASE (which is likely pretty old version by now).

What will happen if we simply remove the CSS completely? We might break some 8.9 IT, and we can bail out of that one for 8.9 if needed with something like this
https://github.com/SonarSource/sonar-scanner-msbuild/blob/e8e1379419222d45255b6aab50ea1c0abce4be48/its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java#L348

Reply:
https://sonarsource.atlassian.net/browse/SCAN4NET-264